app/helpers/slides_helper.rb

```diff
@@ -41,4 +41,22 @@ def accessible_sign_slides(slide)
     accessible_sign_ids = slide.signs.visible_or_owned_by(current_user).ids
     slide.sign_slides.select { |ss| accessible_sign_ids.include? ss.sign_id }
   end
+
+  # `controls:true muted: true` attributes mute videos automatically and display a controls below video
+  # Source: https://www.w3schools.com/tags/att_video_muted.asp %>
+  def slide_video_tag(slide, field)
+    if current_page?(controller: 'signs', action: 'play')
+      if field == :background
+        video_tag(slide.background, id: "js-background-video", class: "ui-slide-background", preload: "auto", loop: true)
+      else
+        video_tag(slide.foreground, id: "js-foreground-video", class: "ui-slide-foreground", preload: "auto", loop: true)
+      end
+    else
+      if field == :background
+        video_tag(slide.background, controls: true, muted: true, id: "js-background-video", class: "ui-slide-background", preload: "auto", loop: true)
+      else
+        video_tag(slide.foreground, controls: true, muted: true, id: "js-foreground-video", class: "ui-slide-foreground", preload: "auto", loop: true)
+      end
+    end
+  end
```

Comment:
This could be done a little more elegantly:

```
  def slide_video_tag(slide)
    if slide.foreground.present?
      file_name = slide.foreground
      slide_id = 'js-foreground-video'
      slide_class = 'ui-slide-foreground'
    else
      file_name = slide.background
      slide_id = 'js-background-video'
      slide_class = 'ui-slide-background'
    end

    if current_page?(controller: 'signs', action: 'play')
      has_controls = false
      is_muted = false
    else
      has_controls = true
      is_muted = true
    end

    video_tag(file_name, controls: has_controls, muted: is_muted, id: slide_id, 
              class: slide_class, preload: "auto", loop: true)
  end
```

But I'm ok with it because it's clear and easy enough to read.